Makefile

```diff
@@ -460,6 +460,14 @@ yetus:
 		--empty-patch \
 		--plugins=all \
 		--patch-dir=/src/yetus-output
+	@echo Checking Dockerfiles for inconsistencies
+	make -C tools/dockerfile-from-checker/
+	./tools/dockerfile-from-checker/dockerfile-from-checker ./ \
+		-i ./eve-tools/bpftrace-compiler/Dockerfile \
+		-i ./pkg/optee-os/Dockerfile \
```

Comment:
I don't think we should ignore other directories besides bpftracer. It's better to ask for these inconsistencies to be fixed.

Reply:
and block this PR until then?

Reply:
You can just remove the ignoring and make it through errors every time so that the people will notice the issue =)

Reply:
But then yetus will fail on every PR.

Reply:
Argh... I didn't notice it's a part of Yetus target... Nope, I think it should be a dedicated target.

Reply:
What about renaming `yetus` to `lint`? I don't want to go through all the hazzle of creating a new GH action ...

Reply:
As @rene mentioned, the Yetus target is not called by any GH action. So, if we want to add it to GH actions, we have to add a new action anyway. Just don't do it if you don't want to. We can do it later.

Reply:
Ah, I see. Can I add it back to the `make test` target?

Reply:
For local usage, sure. I would still like not to ignore the directories that do not have explicit requirements to be ignored. But it's also not an original scope of this PR and can be fixed later if you want.

Reply:
Thx, now I understand.

Reply:
done: https://github.com/lf-edge/eve/actions/runs/11292579668/job/31408847842?pr=4334